superset-frontend/src/explore/controls.jsx

```diff
@@ -83,6 +83,8 @@ export const PRIMARY_COLOR = { r: 0, g: 122, b: 135, a: 1 };
 // input choices & options
 export const D3_FORMAT_OPTIONS = [
   ['SMART_NUMBER', 'Adaptative formating'],
+  [' ', 'Original number'],
+  [',d', ',d (12345.432 => 12,345)'],
```

Comment:
How about changing the description 

`,d (12345.432 => 12,345)` to `Integer (12345.432 => 12,345)`

(optional) There are also constants in the package.

```ts
import { NumberFormats } from '@superset-ui/number-format';
NumberFormats.INTEGER
NumberFormats.SIGNED_INTEGER
```

Reply:
Some languages uses "," for decimal points, so it's probably better for i18n by being more specific.